docs/understanding-airbyte/jobs.md

```diff
@@ -11,14 +11,46 @@ Thus, there are generally 4 types of workers.
 
 **Note: Workers here refers to Airbyte workers. Temporal, which Airbyte uses under the hood for scheduling, has its own worker concept. This distinction is important.**
 
-## Job State Machine
+## Sync Jobs
 
-Jobs have the following state machine.
+At a high level, a sync job is an individual invocation of the Airbyte pipeline to synchronize data from a source to a destination data store.
+
+### Sync Job State Machine
+
+Sync jobs have the following state machine.
 
 ![Job state machine](../.gitbook/assets/job-state-machine.png)
 
 [Image Source](https://docs.google.com/drawings/d/1cp8LRZs6UnhAt3jbQ4h40nstcNB0OBOnNRdMFwOJL8I/edit)
 
+### Attempts and Retries
+
+In the event of a failure, the Airbyte platform will retry the pipeline. Each of these sub-invocations of a job is called an attempt.
+
+### Retry Rules
+
+Based on the outcome of previous attempts, the number of permitted attempts per job changes. Currently, Airbyte is configured to allow the following:
+
+* 5 subsequent attempts where no data was synchronized
+* 10 total attempts where no data was synchronized
+* 10 total attempts where some data was synchronized
```

Comment:
nit: probably useful to add an example for illustration here.

Reply:
probably after the retry backoff section so we cover the entire feature fully.

Reply:
Any ideas on what kind of visualization would work best? All I can think of is a table

Reply:
a table seems fine to start.

the main thing is to be explain when backoff happens and how the configured variable affect how many failed attempts and total attempt in a complicated case.

Reply:
I'll take a pass at a table